### DISCUSSION TITLE
Multilang support

### DISCUSSION THREAD

**Discussion by fmendezy** [zidoro/pomatez#480] [Ideas]
Hi everyone,
I would like to volunteer to translate the application to Spanish and considering that probably other people in the future will want to add their languages, I would like to ask the developers to incorporate the multilanguage option, please count on me from now on to translate to Spanish.

Thank you!
fmendezy

**Comment by sekwah41** [comment]
Great idea! Is it ok if I convert this to an issue so we can properly track this?

**Reply by fmendezy** [reply]
Its ok

**Reply by sekwah41** [reply]
Ah ok, so ive made a new issue for it, it seems that issues can be converted to discussions but you can only create issues from discussions and link them.

Itll properly be tracked in https://github.com/zidoro/pomatez/issues/481
